wiki/Article_Styling_Criteria/en.md

```diff
@@ -82,150 +103,232 @@ tags:
+#### See also
 
-#### Titles
+See also hatnotes are used to suggest a reader other points of interest from a given article. They must start with the words `See also` followed by a colon (`:`). For example:
 
-**All articles must have one level 1 heading, being the article's title.**
+```
+## Beatmapping
 
-Article titles are to be in title case and **should never contain styles, links, nor images.** The title of an article is the name of the folder that the article is located in.
+*See also: [Storyboarding](/wiki/Storyboarding)*
 
-The level 1 heading should be placed at the start of the article unless you have reference links or metadata placed there. If this is the case, place the level 1 heading underneath it.
+Beatmapping is the process of a creator creating a beatmap. This process includes: selecting music, timing and mapping the beatmap, and testing the beatmap. 
+```
 
-For English article titles, if you need to reword the title, you must rename the folder to match the article title. There are certain exceptions to this, however.
+#### For see
 
-#### Sections
+For see hatnotes are similar to see also, but may help reduce ambiguation and are generally more descriptive. They must start with the word `For` followed by the description, a comma, `see`, then a colon (`:`). For example:
 
-All section headings are to be using levels 2 to 5 headings.
+```
+# Installation
 
-Section headings are to use sentence case and **should never contain styles nor links**. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading).
+*For the Linux installation, see: [Installation/Linux](/wiki/Installation/Linux)*
+```
 
-Unlike titles, section headings can have small image icons.
+#### Notice
 
-All sections **must** contain some text, even if they are containers for subsections. For example:
+A notice can be placed where appropriate in a section, but must start off the paragraph and use italics. Notices may contain bolding where appropriate, but be kept to a minimum. These must begin with "Note", "Notice", "Caution", or "Warning" followed by a colon (`:`). These are written as complete sentences, thus, unlike the other hatnotes, must use a full stop or exclamation mark, if appropriate. Anything within the same paragraph of a notice must be italicised along with the notice.
 
 ```
-# Game Modes
+*Note: You will need a decent internet connection to connect to Bancho and submit scores.*
+```
 
-## ![](/wiki/shared/mode/osu.png) osu!
+Note is something that the reader should be made aware about, but only applies a small or specific group of readers. Notice is for a more direct warning that the reader should be aware of. Caution is to warn the reader about unintended consequences. Warning is to warn the reader that there may be action aginst them.
```

Comment:
No `code markup` or even "quotations" on the key terms. Basically no emphasis on the terms.

Reply:
Can you elaborate?

Reply:
`Note`, `Notice`, etc. should stand out in this paragraph.